### ISSUE TITLE
NameError: name 'match_kg_i' is not defined. Did you mean: 'match_kg'?

### ISSUE THREAD

**Issue by fantastic-413** [David-Li0406/DALK#1]
When you run `python MindMap_revised.py` (the readme says `python MindMap.py`, but there is no such file), you will get an error:
![image](https://github.com/user-attachments/assets/761e2e09-de94-4eb5-bb13-dcc61a5d0e33)
Because `match_kg_i` is not initialized in the `MindMap_revised.py` file, initialization needs to be added.
![image](https://github.com/user-attachments/assets/e5e7ea13-d1ed-4442-a290-fcf1f584d91a)

**Comment by David-Li0406** [comment]
Yes, there should be a line "match_kg_i = entity_embeddings["entities"][max_index]" to initialize match_kg_i, which I have added to the current file. Thanks for this kind reminder.
